packages/web-runtime/src/components/UserMenu.vue

```diff
@@ -114,6 +203,12 @@ export default {
     })
   },
   methods: {
+    byteToGb(value) {
```

Comment:
Pretty sure we handle this with an external library or a helper somewhere, let me check

Reply:
`filesize`

Reply:
ok

Reply:
for what it's worth, there is a `OcResourceSize` component in ODS which handles all the conversion (uses `filesize` internally as well) and creates a span if I remember correctly.